### DISCUSSION TITLE
Allow temperature sensors to report temperatures in K

### DISCUSSION THREAD

**Discussion by emontnemery** [home-assistant/architecture#862] [Entity Models] [answered]
Proposal:
- Add `K` as a valid unit for temperature sensors

Background:
- `K` is a valid temperature unit, and we do allow the user to override display temperature to `K` since a few releases back

Related core PR:
- https://github.com/home-assistant/core/pull/86073

**Comment by frenck** [comment, marked as answer]
> and we do allow the user to override display temperature to K since a few releases back

Yeah, that kinda sets the need for it actually. It maybe shouldn't have slipped in there, but that happened.

I'm not sure about the practical use, but since we have it out there: I'm not against adding it.

../Frenck
